packages/admin/src/app/shared/schemas/user.schema.ts

```diff
@@ -30,6 +30,16 @@ export const UserUpdateSchema = UserSchema.extend({
   roleId: z.string().uuid().optional(),
 }).strict();
 
+// for registering a new user
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export const UserRegisterSchema: any = z
```

Comment:
Can you try removing the `: any` and the the eslint-disable line and see if it works and you don't get an typescript warning.
We should only need `any` when we have lazy loaded schemas inside the schema you are defining.
I've found, we don't declare `any` IDEs handle typing analysis a little better.  If linting complains anyway by removing then the `any` is totally acceptable in this instance.

Reply:
fixed

Reply:
removing the any part causes an error on typing due to the form values possibly being undefined
to get around it, i added nullable and optional to the schema